public/plugin.ts

```diff
@@ -64,6 +65,9 @@ import {
   OBSERVABILITY_EMBEDDABLE_ID,
 } from './embeddable/observability_embeddable';
 import { ObservabilityEmbeddableFactoryDefinition } from './embeddable/observability_embeddable_factory';
+import { coreRefs } from './framework/core_refs';
+import { S3DataSource } from './framework/datasources/s3_datasource';
```

Comment:
Looks like a double import is causing CI failure.

Reply:
thanks, didn't get time to test yesterday